pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/constants/RouteConstants.java

```diff
@@ -102,6 +102,8 @@ public static final class API {
                 PathParam.USER_GUID, PathParam.STUDY_GUID, PathParam.INSTANCE_GUID);
         public static final String USER_ACTIVITY_UPLOADS = fmt(BASE + "/user/%s/studies/%s/activities/%s/uploads",
                 PathParam.USER_GUID, PathParam.STUDY_GUID, PathParam.INSTANCE_GUID);
+        public static final String USER_ACTIVITY_DOWNLOADS = fmt(BASE + "/user/%s/studies/%s/activities/%s/questions/%s/downloads",
```

Comment:
Should this be **download** instead of **downloads**?  Should there be a one to many relationship implied in the path?

Reply:
user might need to do multiple downloads. As of now one at a time only though. Just went with similar to current "uploads" API in DSS.  I am ok to change if you prefer it.

Reply:
Does the code expect a one:many relationship?  If so, I'm okay with the plural and by convention it will be 1:1.

Reply:
Each call only one signed url. will change it.

Reply:
updated